### DISCUSSION TITLE
Тул для замены текстурки пропа с любой стороны.

### DISCUSSION THREAD

**Discussion by Dawaren** [octothorp-team/public-dobrograd#746] [Предложения]
**Всем привет!**

Буду краток, я наверное много прошу, но я хотел бы что бы сделали тул, с помощью которого можно поменять текстурку не всего пропа, а только с одной стороны.

**Зачем?**
Это будет наиполезнейший тул для строителей, не надо будет к примеру ставить две стены, и с одной стороны ставить одну текстурку, а на другой стене другую текстурку.

**Жду ваших голосов!**

**Comment by Kyper1** [comment]
О Боже. Ты только, что написал о том, о чем я мечтал всю свою жизнь.
